kde-frameworks/kdelibs4support/kdelibs4support-5.109.0.ebuild

```diff
@@ -88,7 +93,10 @@ src_prepare() {
 	ecm_src_prepare
 
 	if ! use handbook; then
-		sed -e "/kdoctools_install/ s/^/#DONT/" -i CMakeLists.txt || die
+		sed -e "/kdoctools_install/ s/^/#DONT/;/find_package(KF5DocTools/ s/^/#DONT/" -i CMakeLists.txt || die
```

Comment:
Things like this are likely going to benefit from a comment, or even better, a patch which can be sent upstream.

Reply:
OK, fair enough. Just note that in the commit message then.